### ISSUE TITLE
Training stopped after 5 epochs while Semimtr finetuning

### ISSUE THREAD

**Issue by wulinunu** [amazon-science/semimtr-text-recognition#7]
A problem occurred when trying to train the Semimtr:

The training stage will automatically stop after 5 epochs no matter what value of epoch I set in the config file 'configs/semimtr_finetune.yaml'.

However, if I set epoch to less than 5, then everything seems to work just fine.

**Comment by aaberdam** [comment]
You should also increase the periods of the scheduler in the configuration file. By default, they set to be: `[ 3, 1, 1 ]`.

Let me know if it works for you.

**Comment by wulinunu** [comment]
Oh I see. That works.
Thank you!
